docs/monitor/view-active-alerts.md

```diff
@@ -19,52 +20,45 @@ From this tab, you can also filter alerts with the right hand bar. More specific
+  - Filter the alerts based on the nodes of your War Room.
 
-## View alert details
+### View alert details
 
 By clicking on the name of an entry of the table you can access that alert's details page, providing you with:
 
 - Latest and Triggered time values
 - The alert's description
-- A link to the Community forum's alert page
+- A link to the Netdata Advisor's page about this alert 
 - The chart at the time frame that the alert was triggered
-- The alert's information: Node name, chart ID, type, component and class
+- The alert's information: Node name, chart instance, type, component and class
 - Configuration section
 - Instance values - Node Instances
 
-![image](https://user-images.githubusercontent.com/70198089/226339928-bae60140-0293-42cf-9713-ac4901708aba.png)
-
-At the bottom of the panel you can click the green button "View dedicated alert page" to open a [dynamic tab](https://github.com/netdata/netdata/blob/master/docs/quickstart/infrastructure.md#dynamic-tabs) containing all the info for this alert in a tab format, where you can also run correlations and go to the node's chart that raised the particular alert.
+At the bottom of the panel you can click the green button "View alert page" to open a [dynamic tab](https://github.com/netdata/netdata/blob/master/docs/quickstart/infrastructure.md#dynamic-tabs) containing all the info for this alert in a tab format, where you can also run correlations and go to the node's chart that raised the particular alert.
 
-![image](https://user-images.githubusercontent.com/70198089/226339794-61896c35-0b93-4ac9-92aa-07116fe63784.png)
+### Silence an alert
 
-<!-- 
-## Local Netdata Agent dashboard
+From this tab, the "Silencing" column shows if there is any rule present for each alert, and from the "Actions" column you can create a new [silencing rule](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/notifications.md#silencing-alert-notifications) for this alert, or get help and information about this alert from the [Netdata Assistant](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-assistant.md).
 
-Find the alerts icon ![Alerts
-icon](https://raw.githubusercontent.com/netdata/netdata-ui/98e31799c1ec0983f433537ff16d2ac2b0d994aa/src/components/icon/assets/alarm.svg)
-in the top navigation to bring up a modal that shows currently raised alerts, all running alerts, and the alerts log.
-Here is an example of a raised `system.cpu` alert, followed by the full list and alert log:
+## Alert Configurations tab
 
-![Animated GIF of looking at raised alerts and the alert
-log](https://user-images.githubusercontent.com/1153921/80842482-8c289500-8bb6-11ea-9791-600cfdbe82ce.gif)
+From this tab you can view all the configurations for all potential alerts in your War Room. Each row concerns one alert, and it provides information about it in the rest of the table columns.
```

Comment:
```suggestion
From this tab you can view all the configurations for all running alerts in your War Room. Each row concerns one alert, and it provides information about it in the rest of the table columns.

By running alerts we mean alerts that are related to some metric that is or was collected. Netdata may have more alerts pre-configured that aren't applicable to your monitoring use-cases.

```

this came from a chat I was with Costa and @kapantzak, where he made it clear this isn't all the pre-configured alerts just those related to things Cloud is aware of (that were or are being monitored)

pinging @sashwathn in case he has more to share

Note: we will be changing the tab names as per a suggestion from Costa but not sure when they will be done, so we will need to review this then cc/ @kapantzak

Reply:
If we change the tab names, let's make sure the documentation gets the proper updates _the same day_

I'm doing this arrangement in the Dashboard sections so it is easier to pinpoint what to update. Ideally the one who merges the PR for the tab changes should in turn go and update related docs around it